src/types.rs

```diff
@@ -125,9 +125,14 @@ pub enum LakeError<E> {
         #[from]
         error: aws_sdk_s3::error::SdkError<E>,
     },
-    #[error("Failed to convert integer")]
+    #[error("Failed to convert integer: {error}")]
     IntConversionError {
         #[from]
         error: std::num::TryFromIntError,
     },
+    #[error("AWS Smithy byte_stream error: {error}")]
```

Comment:
```suggestion
    #[error("AWS Smithy byte_stream error: {error:?}")]
```

Can we print the debug message, the normal display message doesn't provide much information. Same for `AwsError` too please 🙏🏼